packages/integration-tests/src/__tests__/integration/mssql/__scenarios.ts

```diff
@@ -6,8 +6,8 @@ export const scenarios = [
     name: 'findUnique where PK',
     up: `
         create table teams (
-          id int primary key not null,
-          name varchar(32) not null unique
+          id int CONSTRAINT PK__teams__CustomNameToAvoidRandomNumber primary key not null,
+          name varchar(32) not null CONSTRAINT UQ__teams__CustomNameToAvoidRandomNumber unique
```

Comment:
This is ugly :(

Reply:
Might be better to use our default constraint name here, so this is not rendered in the resulting Prisma Schema at least?

Reply:
Feel free to edit all this file and put the defaults 😅 It's fun

Reply:
Points to a serious smell in our implementation :/ (or just in the reality of using SQL Server 🐑 )